MANIFEST.in

```diff
@@ -0,0 +1 @@
+include version.h
```

Comment:
setup.py parses it to pull the version number (and unfortunately does this at runtime rather than when the package is generated...).  I want to limit the location where the version number is set to once place and using a regex to grab it from a few defines is a lot simpler than autogenerating a bunch of stuff when I build.